NineChronicles.Headless/GraphTypes/States/StateContext.cs

```diff
@@ -1,21 +1,99 @@
 using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
 using Bencodex.Types;
 using Libplanet;
 using Libplanet.Action;
 using Libplanet.Assets;
 
 namespace NineChronicles.Headless.GraphTypes.States
 {
-    public class StateContext
+    public class StateContext : IAccountStateDelta
```

Comment:
This is just to be able to use the extension methods available from IAccountStateDelta such as GetSheets, TryGetAvatarStateV2, TryGetArenaAvatarState and GetEnemyAvatarState.  If those extension methods can be used from a different type, then this could be removed.